src/components/VAudioTrack/VAudioTrack.vue

```diff
@@ -1,10 +1,11 @@
 <template>
-  <div
+  <Component
+    :is="isBoxed ? 'VLink' : 'div'"
     class="audio-track group"
-    :aria-label="$t('audio-track.aria-label')"
+    :aria-label="ariaLabel"
     role="region"
     v-bind="layoutBasedProps"
-    v-on="layoutBasedListeners"
+    @keydown.native="handleKeydown"
```

Comment:
We probably cannot use an event handler like this on a `div`. Besides, we would probably need to add a `tabindex` if a div has an keydown handler. Or is it a requirement only for a click handler? This is a warning I get  on a single audio page:
```
[Vue warn]: The .native modifier for v-on is only valid on components but it was used on <div>.
found in
---> <VAudioTrack> at src/components/VAudioTrack/VAudioTrack.vue
```

Maybe we can add a keydown handler with v-bind, in addition to the `layoutBasedProps`?

Reply:
I couldn't figure out how to add the `.native` modifier when using `v-on`, the event handler never got applied to the element when I tried this for example:

```js
const eventHandlers = computed(() => {
  if (!isBoxed.value) return {}

  return {
    'keydown.native': ...
  }
})
```

Is there a different way to do it? As far as I can tell modifiers aren't available for the `v-on` object syntax: https://github.com/vuejs/vue/issues/7846

Reply:
Regarding the div problem, the div won't be focusable ever, there's no tabindex, so the key handler will just be ignored, right? And in those cases, indeed it _should_ be ignored as the event handler itself will just do an early return anyway.

Is there a way to suppress that vue warning? In this case I think it is safe to ignore considering that it doesn't appear to be possible to conditionally bind a `.native` modified event handler.